docs/run/primitives.mdx

```diff
@@ -51,21 +51,112 @@ For hardware providers, implementing primitives natively means you can provide y
 way to access your hardware features. It is therefore easier for your users to benefit from your hardware's
 best capabilities.
 
+## V2 primitives
+
+Version 2 is first the major interface change since the introduction of Qiskit Runtime primitives. Based on user feedback, the new version introduces the following major changes:
+
+* The **new interface** lets you specify a single circuit and multiple observables (if using Estimator) and parameter value sets for that circuit. 
+* You can **turn on or off individual error mitigation and suppression methods**.
+* You can choose to **get counts or per-shot measurements** instead of quasi-probabilities from V2 Sampler.
+* Due to scaling issues, V2 Sampler does not support specifying a resilience level, and V2 Estimator supports only levels 0 - 2. 
+* To reduce the total execution time, v2 primitives only support ISA circuits. If you need more advanced transpilation, you can use the [Qiskit transpiler](advanced-runtime-options#runtime-compilation) (manually) or the [AI-driven transpilation service (premium users)](../transpile/qiskit-transpiler-service).
```

Comment:
Slight change in wording as the original sounded like transpilation was optional. 

```suggestion
* To reduce the total execution time, v2 primitives only support ISA circuits. You can use the [Qiskit transpiler](advanced-runtime-options#runtime-compilation) (manually) or the [AI-driven transpilation service (premium users)](../transpile/qiskit-transpiler-service) to transform the circuits before making the queries to the primitives.
```